csrc/expr_evaluator.h

```diff
@@ -60,6 +60,13 @@ class TORCH_CUDA_CU_API ExpressionEvaluator {
     return precomputed_values_;
   }
 
+  //! Agument the evaluator with the exact root-domain map such that
```

Comment:
Nit: Augment
This could be done with the almost exact map instead of exact

Reply:
We could, but needs some extension since at this point the fusion has symbolic IDs, and there's no expr connecting the root and rfactor domains of symbolic reshape ops, so the graph traversal would need to gracefully handle symbolic fusions in a best-effort fashion.